r/tests/testthat/test-dataset.R

```diff
@@ -1767,3 +1767,13 @@ test_that("Dataset writing: unsupported features/input validation", {
     write_dataset(ds, tempfile(), basename_template = NULL)
   )
 })
+
+# see https://issues.apache.org/jira/browse/ARROW-11328
+test_that("Collecting zero columns from a dataset doesn't return entire dataset", {
+  tmp <- tempfile()
+  write_dataset(mtcars, tmp, format = "parquet")
+  expect_equal(
+    open_dataset(tmp) %>% select() %>% collect() %>% dim(),
+    c(32,0)
```

Comment:
Space after comma (https://style.tidyverse.org/syntax.html#commas)
```suggestion
    c(32, 0)
```